src/helpers/volunteerHelper.ts

```diff
@@ -105,13 +105,13 @@ export const volunteerSubtitle = (
   let date = eventDate(
     volunteerListDate(volunteer, withDate),
     undefined,
-    isSectioned ? 'HH:mm' : 'DD.MM.YYYY HH:mm'
+    isSectioned ? 'HH:mm [Uhr]' : 'D. MMMM YYYY[,] HH:mm [Uhr]'
```

Comment:
can you write the difference between these formats before and after?

Reply:
|before|after|
|---|---|
|12:00|12:00 Uhr|
|04.05.2022 12:15|4. Mai 2022, 12:15 Uhr|